packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -379,4 +397,41 @@ class AndroidCameraCameraX extends CameraPlatform {
     return Preview(
         targetRotation: targetRotation, targetResolution: targetResolution);
   }
+
+  /// Configures and starts a video recording.
+  @override
+  Future<void> startVideoRecording(int cameraId,
+      {Duration? maxVideoDuration}) async {
+    assert(cameraSelector != null);
+    assert(processCameraProvider != null);
+
+    processCameraProvider ??= await ProcessCameraProvider.getInstance();
+    recorder = Recorder(); // TODO(gmackall): Configure resolution info here.
+    videoCapture = await VideoCapture.withOutput(recorder!);
+    processCameraProvider!.bindToLifecycle(cameraSelector!, [videoCapture!]);
+    videoOutputPath = await SystemServices.getTempFilePath('MOV', '.mp4');
```

Comment:
The main reason I wrapped it was because the video capture classes don't return the path at any point (only take it in a constructor). 

So given that it had to be returned from the dart side, it felt easiest to wrap the initial creation of the path on the dart side and then store till returning.

Reply:
That makes sense! We can modify image capture to use the wrapped method in the future. I can file an issue

Reply:
Consider making "MOV" A named constant. 
Can you link to the documentation that says we will get an mp4 here or if the file extension does not matter pick something that does not imply a format like "temp"

Reply:
From [this page](https://developer.android.com/training/camerax/video-capture#bind-videocapture), the Recorder "selects the most suitable format for the system," so it looks like we are not guaranteed an mp4. Will change to temp.